active/PagerControl/PagerControl.md

```diff
@@ -0,0 +1,336 @@
+
+### GamePad 
+* Pager Control will not have engagement. The internal components (ComboBox and NumberBox) will keep their current engagement and accessibility features for game pad. 
+* The user can use the right and left arrow keys to navigate between the buttons and use the A button to select an item. 
+* Guidance will be added for app developers that are targeting Xbox applications to not place items to the left or right of the pager control. 
+        * Text added to guidance: “If you are developing an application that will target Xbox, we recommend not placing any UI elements to the left or right of the pager control so the end user does not have to navigate through the pager control in order to reach the next UI element.” 
+
+### Localization
+* The numbers will follow standard behavior when localized to/from a language that uses a numeral system that differs from Western Arabic. 
+* The prefix and suffix text will follow standard mirroring behavior when localized to/from languages that are RTL or LTR. 
+
+## Data and Intellegence Metrics 
+
+Adoption and validation of the control in the community
+* Measurement: Use telemetry to determine how many first and third party developers are using the control in preview. 
+* Measurement: Count of posts providing feedback to us through Github and Discord. 
+
+Questions that can be answered through telemetry metrics to help focus future features and improvements: 
+Which display mode is the most commonly used? 
+* Measurement: Count of how many applications include the pager control type broken out by display mode.
+
+How often are developers using this control with an indefinite number of pages?
+* Measurement: Count of how many applications include the pager control type and have the NumberOfPages property empty.
+
+
+## Open Questions 
+* Should the combo box mode be an editable combo box? My worry with making it an editable combo box is the validity of having the number box mode. Would it hurt to have both editable combo box and number box as options?
+* I use the terms index and page throughout this document and the APIs. The APIs are all index while the documentation is mostly using page. Should I update the APIs to use page instead of index?
+* There are certain APIs that are specific to one of the three display modes and they will be ignored if the display mode chosen does not support those APIs. Is that something that needs to be reconsidered so it does not cause confusion for the developers?
+* The enum for the edge button visibility has the four values Auto, AlwaysVisible, HiddenOnEdge, and None. Do those names make sense and translate to the intended behavior? Does the enum name make sense?
+* The ButtonPanelShowFirstAndLast property will show or hide both the first and last page in the numerical button panel. Should this API be split in to two so the developer can show or hide the first and last button individually?
```

Comment:
I think there is hardly a use case to only show one of the button panels except when you reach either end of the page list. Splitting this up would also increase the API surface and introduce more properties that are not relevant at all times, so I think we should not split this up.